app/views/sandbox/budget_guide.html.erb

```diff
@@ -0,0 +1,269 @@
+<div class="column">
+
+  <main class="content">
```

Comment:
We have a white space between the header and the gradient. This is the way it's supposed to be?

<img width="490" alt="captura de pantalla 2017-07-12 a las 13 25 23" src="https://user-images.githubusercontent.com/24740/28115523-999adc3a-6705-11e7-9ead-052a20174c3c.png">